e2e/support/cypress_sample_instance_data.js

```diff
@@ -14,6 +14,11 @@ export const ORDERS_QUESTION_ID = _.findWhere(SAMPLE_INSTANCE_DATA.questions, {
   name: "Orders",
 }).id;
 
+export const ORDERS_QUESTION_ENTITY_ID = _.findWhere(
```

Comment:
I added these exports to be able to re-use these resources when possible instead of having to create new ones in the tests